design/ceph/object-store.md

```diff
@@ -72,8 +71,11 @@ The pools are the backing data store for the object store and are created with s
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  preservePoolsOnDelete: True
 ```
 
+If there is an `zone` section in object-store configuration, then the pool section in the object-store-zone resource will be used to define the pools.
+
 ### Gateway
 
 The gateway settings correspond to the RGW service.
```

Comment:
* is the operator expected to create the k8s service as well? if so, it may be that more parameters are needed
* is there a usecase for rgw to ceph affinity (e.g. running rgw and osd on the same node)?

Reply:
> * is the operator expected to create the k8s service as well? if so, it may be that more parameters are needed

Yes, the operator creates the RGW daemons and k8s service today. I'm assuming the same will apply to multi-site design. More `gateway` parameters are certainly a possibility.

> * is there a usecase for rgw to ceph affinity (e.g. running rgw and osd on the same node)?

There's no affinity between rgw and osd daemons.

Reply:
A small note, each operator creates the RGW daemons and service that will be running on the local k8s cluster (this is for a case of geo-replication i.e two k8s cluster or more in different Geo-location)